candidate_base.go

```diff
@@ -308,6 +308,10 @@ func (c *candidateBase) close() error {
 func (c *candidateBase) writeTo(raw []byte, dst Candidate) (int, error) {
 	n, err := c.conn.WriteTo(raw, dst.addr())
 	if err != nil {
+		// If the connection is closed, we should return the error
+		if errors.Is(err, io.ErrClosedPipe) {
+			return n, err
+		}
 		c.agent().log.Infof("%s: %v", errSendPacket, err)
```

Comment:
Just wondering why this is using the info level? Shouldnt we print this at least as a warning?

Reply:
For ref: https://github.com/m1k1o/pion-ice/commit/8c1adf18f14735ea6c35e8a714392f509f84957d